### ISSUE TITLE
`arguments` keyword not being matched

### ISSUE THREAD

**Issue by smaili** [atom/language-javascript#189]
Screenshot:

![pic](https://cloud.githubusercontent.com/assets/4451986/9065608/87788f7c-3a87-11e5-9f29-9c1cc96a3583.png)

**Comment by Victorystick** [comment]
Not an issue as such. `arguments` isn't a keyword. https://people.mozilla.org/~jorendorff/es6-draft.html#sec-keywords

**Comment by smaili** [comment]
@Victorystick, that's a reference to the ES6 draft.  I believe ES5 treats `arguments` as a keyword, or at the very least a reserved word.

**Comment by Victorystick** [comment]
Nope. ES5 spec: https://es5.github.io/#x7.8.4

ES6 didn't remove anything.

**Comment by mehcode** [comment]
Shouldn't `arguments` at least be a `support.*` something or other? It might not be a keyword but its certainly a "language feature".

**Comment by smaili** [comment]
I stand corrected.  They seem to refer to it as an "identifier" :)

In any case, I don't see anything wrong with having some kind of group matching to at least give highlighters the opportunity to stylize it if they so choose.

**Comment by smaili** [comment]
Maybe also add `constructor` and `prototype` to the list?

**Comment by 50Wliu** [comment]
> constructor

Please see #191 for more details (basically, it's already tokenized, but some themes don't colorize it).

**Comment by smaili** [comment]
@50Wliu You seem to be on to something.  The same happens to `delete`.

![delete](https://cloud.githubusercontent.com/assets/4451986/9139581/f350f3c4-3cdf-11e5-9b75-6d0d00fc2337.png)

**Comment by YuriSolovyov** [comment]
I agree that `arguments` need some more specialized highlighting, it would make it easier to spot it in code.

**Comment by 50Wliu** [comment]
All right.  Coming back to this: Does anyone have any suggestions for what `arguments` should be tokenized as?

**Comment by YuriSolovyov** [comment]
@50Wliu IMO same as `this` - kind of metavariable.

**Comment by MaximSokolov** [comment]
`variable.language.arguments`
